update status correction docs (#25267)
DataDog/documentation: content/en/dashboards/guide/slo_data_source.md

| Burn rate | Metric-based or Time Slice | For each time bucket, the burn rate shows the observed error rate divided by the ideal error rate. | The burn rate over the widget's time frame. |
| Error budget burndown | Metric-based or Time Slice | The error budget burned over time. It starts at 100% (unless there were bad events/minutes within the first time bucket) and decreases with bad events/minutes. | Error budget burndown is not available in scalar widgets. |

#### SLO status corrections

> For SLOs that have [status corrections][2], you can decide whether you want to enable or disable corrections in charts using the SLO data source. By default, corrections will be enabled and applied to the chart. You can update the behavior for displaying corrections via the advanced options menu options:

Reviewer: ```suggestion
For SLOs that have [status corrections][2], you can enable or disable corrections in charts using the SLO data source. By default, corrections are enabled and applied to the chart; you can disable with the advanced options menu options:
```

> {{< img src="dashboards/guide/slo_data_type/slo-data-source-correction.gif" alt="Graph editor configuration with the slo data type selected and the good events measure selected" style="width:100%;" >}}

Reviewer: This needs to be an MP4 file. Let me know if you have any questions on this!
